tools/FakeItEasy.PrepareRelease/Program.cs

```diff
@@ -102,10 +102,18 @@ public static async Task<IList<Issue>> GetIssuesInMilestone(this IGitHubClient g
 
         public static IEnumerable<string> GetIssuesReferencedFromRelease(Release release)
         {
+            // Release bodies should contain references to fixed issues in the form 
+            // (#1234), or (#1234, #1235, #1236) if multiple issues apply to a topic.
+            // It's hard (impossible?) to harvest values from a repeated capture group,
+            // so grab everything between the ()s and split manually.
             var issuesReferencedFromRelease = new HashSet<string>();
-            foreach (Match match in Regex.Matches(release.Body, @"\(#(<issueNumber>[0-9]+)\)"))
+            foreach (Match match in Regex.Matches(release.Body, @"\((?<issueNumbers>#[0-9]+((, )#[0-9]+)*)\)"))
             {
-                issuesReferencedFromRelease.Add(match.Groups["issueNumber"].Value);
+                var issueNumbers = match.Groups["issueNumbers"].Value;
+                foreach (var issueNumber in issueNumbers.Split(new char[] { '#', ' ', ',' }, StringSplitOptions.RemoveEmptyEntries))
+                {
+                    issuesReferencedFromRelease.Add(issueNumber);
+                }

```

Comment:
Nice touch!

Reply:
Oh? Thanks. I was frustrated that I couldn't just suck the values out with a regex. Ah well, it works. Probably overkill for Sprache!